packages/application-shell/src/components/custom-view-flop-flip-provider/custom-view-flop-flip-provider.tsx

```diff
@@ -0,0 +1,37 @@
+import { ReactNode, useMemo } from 'react';
+import memoryAdapter from '@flopflip/memory-adapter';
+import { ConfigureFlopFlip } from '@flopflip/react-broadcast';
+import { TFlags } from '@flopflip/types';
+import { TFetchLoggedInUserQuery } from '../../types/generated/mc';
+
+type TCustomViewFlopFlipProviderProps = {
+  projectKey?: string;
+  user?: TFetchLoggedInUserQuery['user'];
+  flags?: TFlags;
+  children: ReactNode;
+};
+
+function CustomViewFlopFlipProvider(props: TCustomViewFlopFlipProviderProps) {
+  const adapterArgs = useMemo(
+    () => ({
+      user: {
+        kind: 'user',
+        key: props.user?.id,
```

Comment:
I think the kind isn't needed. That's mainly LD adapter context related. If we want to be minimal we can pass the user and a key as a stable identifier. That to my memory is the basic requirement to match the TBaseUser in flopflip.

Reply:
Updated here: 1db66771d812b230663c074732580405a3cc8b58